lib/adapter.js

```diff
@@ -37,74 +41,83 @@ class CucumberAdapter {
         this.capabilities = capabilities
 
         this.cucumberOpts = Object.assign(DEFAULT_OPTS, config.cucumberOpts)
```

Comment:
@lgandecki 

`
        // convert require path to files
        this.cucumberOpts.require = this.cucumberOpts.require.map(function (path) {
            if (path.match(/.*\*\*\/\*.js/)) return glob.sync(path, {cwd: process.cwd()})

            if (path.match(/.*\.js/)) return path

            return glob.sync(path + '/**/*.js', {cwd: process.cwd()})
        }).reduce(function (a, b) {
            return a.concat(b)
        }, [])`

You can add something like this after this line.

This is to parse the cucumber require option. Please also add glob to import, and dependency in package.json.

By the way, is there a reason we are not using Cucumber.Cli directly?